dist/common/scripts/scylla_util.py

```diff
@@ -381,14 +384,13 @@ def __init__(self):
     @staticmethod
     def is_azure_instance():
         """Check if it's Azure instance via DNS lookup to metadata server."""
+        p = azure_instance()
```

Comment:
creating instance might trigger code you don't want to run on a machine that is not azure
remove and rather make later pointers static